scripts/llnl/common_build_functions.py

```diff
@@ -288,19 +295,20 @@ def build_and_test_host_config(test_root, host_config, report_to_stdout=False, e
     print("[starting unit tests]")
     print("[log file: %s]" % tst_output_file)
 
-    tst_cmd = "cd %s && make CTEST_OUTPUT_ON_FAILURE=1 test ARGS=\"--no-compress-output -T Test -VV -j8\"" % build_dir
-
-    res = sexe(tst_cmd,
-               output_file = tst_output_file,
-               echo=True)
+    if job_count == "":
+        test_job_count=8
+    else:
+        test_job_count=min(job_count, 8)
```

Comment:
```suggestion
    # Use a maximum of 8 job slots for unit tests due to extra parallelism from OpenMP/MPI
    test_job_count=8
    if job_count != ""::
        test_job_count=min(job_count, 8)
```

Reply:
It weirds me out that Python allows variables from inside if blocks to be used....